modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesChannel.java

```diff
@@ -136,9 +136,7 @@ public AndesChannel(FlowControlManager flowControlManager, FlowControlListener l
         this.messagesOnBuffer = new AtomicInteger(0);
         this.flowControlEnabled = false;
 
-        if(log.isInfoEnabled()) {
-            log.info("Channel created with ID: " + getId());
-        }
+        log.info("Channel created with ID: " + getId());
```

Comment:
Log changed in https://github.com/a5anka/andes/commit/644738e63666ed0fbe1f48e4922faa1a2cc6c912